requirements.txt

```diff
@@ -1,2 +1,3 @@
 # requires numpy before it can be installed
 #-e svn://svn.code.sf.net/p/gnuplot-py/code/trunk#egg=gnuplot-py
+git+https://github.com/drjova/flask-iiif.git@personal-fixes # :( :( :(
```

Comment:
It should not be here. Instead add it to https://github.com/inveniosoftware/invenio/blob/master/.travis-devel-requirements.txt.